greeting-service/.openshiftio/application.yaml

```diff
@@ -0,0 +1,173 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: wfswarm-cache-greeting
+  annotations:
+    iconClass: icon-jboss
+    tags: wildfly-swarm, java, microservice
+    template.openshift.io/provider-display-name: "Red Hat, Inc."
+    template.openshift.io/documentation-url: "http://appdev.prod-preview.openshift.io/docs/wf-swarm-runtime.html#mission-cache-wf-swarm"
+    description: >-
+      The Cache Service Mission demonstrates caching with Infinispan.
+parameters:
+- name: SOURCE_REPOSITORY_URL
+  description: The source URL for the application
+  displayName: Source URL
+  value: https://github.com/michalszynkiewicz/wfswarm-cache.git
```

Comment:
Shouldn't it be https://github.com/wildfly-swarm-openshiftio-boosters/wfswarm-cache.git?

Reply:
It sure should, fixed